_config.php

```diff
@@ -1,6 +0,0 @@
-<?php
```

Comment:
Which is better, an empty _config.php file, or a _config folder with an empty config.yml in it? I'm starting to think we need a third "this is a module" identifier.

how about `type: silverstripe/module` in composer? Module manifest does this now doesn't it?